common/src/View/CommandProcessor.cpp

```diff
@@ -258,21 +178,21 @@ namespace TrenchBroom {
             m_lastCommandTimestamp = 0;
         }
 
-        CommandProcessor::SubmitAndStoreResult CommandProcessor::submitAndStoreCommand(UndoableCommand::Ptr command, const bool collate) {
+        CommandProcessor::SubmitAndStoreResult CommandProcessor::submitAndStoreCommand(std::unique_ptr<UndoableCommand>&& command, const bool collate) {
             SubmitAndStoreResult result;
-            result.submitted = doCommand(command);
+            result.submitted = doCommand(command.get());
```

Comment:
So "submit" means "do"? could use a one-line comment giving an example of what the terminology ("submit", "store") means

Reply:
renamed